diff -uNr apt-2.1.11/cmdline/apt.cc apt-2.1.11.mod/cmdline/apt.cc
--- apt-2.1.11/cmdline/apt.cc	2020-10-21 12:53:18.000000000 +0300
+++ apt-2.1.11.mod/cmdline/apt.cc	2020-11-05 16:56:10.361005861 +0200
@@ -101,6 +101,11 @@
 									/*}}}*/
 int main(int argc, const char *argv[])					/*{{{*/
 {
+   //if (getuid() == 0) {
+   //   std::cout << "Ability to run this command as root has been disabled permanently for safety purposes." << std::endl;
+   //   return 1;
+   //}
+
    CommandLine CmdL;
    auto const Cmds = ParseCommandLine(CmdL, APT_CMD::APT, &_config, &_system, argc, argv, &ShowHelp, &GetCommands);
 
